tests/components/myuplink/test_update.py

```diff
@@ -0,0 +1,22 @@
+"""Tests for myuplink update module."""
```

Comment:
Normally this should be a separate PR. But since it's small and just tests it's ok this time.